core/src/mindustry/entities/abilities/RepairFieldAbility.java

```diff
@@ -14,6 +14,7 @@ public class RepairFieldAbility extends Ability{
     public Effect healEffect = Fx.heal;
     public Effect activeEffect = Fx.healWaveDynamic;
     public boolean parentizeEffects = false;
+    public float sameTypeHealMult = 1f;
```

Comment:
Where doc, also put it with the other floats
```suggestion
/** Multiplies healing to units of the same type by this amount. */
    public float sameTypeHealMult = 1f;
```